### ISSUE TITLE
Garamon python bindings on Windows

### ISSUE THREAD

**Issue by laisnegautier** [vincentnozick/garamon#12]
Hello,

I'm not someone with a lot of background in C++ (compiling Garamon using windows was already a hard task), but I would like to know if there is a possibility to make the python binding available for Windows and not Unix users only?

I already used Clifford package, but I need a faster GA backend, which Garamon seems to offer at the end.

Thank you,
Gautier

**Comment by nickmaleki** [comment]
Yes, this is possible. I was able to get the bindings for p10ga working: https://youtu.be/rJuRTZOE99g?si=Pj7nNBUUxnV4CFRp&t=110
